This is a temporary fix for:

37321027 Arabic characters are printing in reverse order

The bug is already fixed upstream. This patch will be removed once the upstream
fix is released and component upgraded.

--- a/charset/pdf.utf-8.simple
+++ b/charset/pdf.utf-8.simple
@@ -30,4 +30,5 @@
 
 0000 04FF ltor single monospace monospace:bold monospace:oblique monospace:bold:oblique
 0500 05FF rtol single monospace
+0600 06FF rtol single monospace
 3000 9FFF ltor double ARPLUmingCN
